sdk/cognitiveservices/Knowledge.QnAMaker/CHANGELOG.md

```diff
@@ -1,6 +1,5 @@
-## 2.1.0-preview.2 (Unreleased)
-
-
+## 3.0.0-preview.1 (2020-10-20)
```

Comment:
Why isn't this 2.1.0-preview.2?

Reply:
This is based on an upgraded swagger which caters to V5.0-preview API. So, bumped up the version here as well, though added customizations to be fully backward compatible from the SDK usage perspective.

Reply:
@AlexGhiondea , please suggest as per sdk / nuget guidance, we will follow.
this is an update form swagger4.0 to swagger 5.0.preview-1
but its totally backward compatible.

Reply:
Discussed offline. Here is the summary

SDK | Swagger   Version | Authoring | Runtime | Comments | Compatibility   - from customer's usage
-- | -- | -- | -- | -- | --
2.0.1 | 4.0 | V1 | V1 | GA | -
2.1.0-preview.1 | 4.0 | V1,V2 | V1, V2 | Backup   plan if 3.0.0 is not possible in Track1 with support for same API for V2   Preview endpoint. This has gaps with what V2preview has to offer our   customers. | Yes
3.0.0-preview.1 | 5.0-preview.1 | V1,V2 | V1 | Since   we got exception, went ahead with new generated files and customizations for   3.0.0-preview.1 in Track1 | Yes (a   few corrections and added functionalities)